plugin/assets/css/src/front-end.css

```diff
@@ -17,17 +17,5 @@
 /*
  * All Frontend CSS.
  */
-@import "./base/variables.css";
-@import "./conf/index.css";
-@import "typography.css";
-@import "./material-components.css";
-@import "./components/contact-form.css";
-@import "./components/masonry-grid.css";
-@import "./components/card.css";
-@import "./components/core.css";
-@import "../../src/block-editor/blocks/common-posts-list/style.css";
-@import "../../src/block-editor/blocks/contact-form/inner-blocks/common/style.css";
-@import "../../src/block-editor/blocks/card/style.css";
-@import "./block/style/masonry.css";
-@import "./overrides.css";
-@import "./tokens/index.css";
```

Comment:
Renamed to `_frontend.css` which includes same files except `material-components.css`